### DISCUSSION TITLE
Is it possible to store the version in localstorage?

### DISCUSSION THREAD

**Discussion by vaynevayne** [mantinedev/mantine#4894] [Q&A]
I don't use 7.x, I want to see the version of 6.x, I need to switch every time, can you store the information of the version I choose in locaoStorage?
<img width="1501" alt="image" src="https://github.com/mantinedev/mantine/assets/124666577/72bcfe4d-a69f-47b7-8a5e-9d3cf96ee0d3">

**Comment by rtivital** [comment]
Just visit the website – https://v6.mantine.dev/
